mkdocs.yml

```diff
@@ -1096,6 +1096,7 @@ nav:
         - integrations/builtin/credentials/medium.md
         - integrations/builtin/credentials/messagebird.md
         - integrations/builtin/credentials/metabase.md
+        - integrations/builtin/credentials/microsoftazuremonitor.md
```

Comment:
We should probably adjust the order slightly so that this is after the generic Microsoft creds.

Reply:
That is a good idea, will tweak shortly.

Reply:
@imchairmanm done